Changelog.md

```diff
@@ -1,6 +1,57 @@
 # Changelog
 This page defines all the changes that were done in the API. There is a difference between the version of the API and the version of Zenya. A new major API version will only be created when we have to introduce breaking changes to the API. New routes, enhanced routes and bug fixes can be introduced with a new Zenya version without creating a new API version.
 
+## Changes in Zenya 6.3.0
+
+Change|Route|Remarks
+|--|--|--|
+Changed|**GET** `/card_files/{card_file_id}/cards/filter/{filter_id}`|Added new querystring parameter include_all_custom_fields, include_field_ids and include_display_name. Removed querystring parameter include_fields.
+Changed|**GET** `/card_files/cards/{card_id}`|Added new querystring parameter include_all_custom_fields, include_field_ids and include_display_name
+Deprecated|**GET** `/card_files/cards/by_qrcode/{qrcode}` | This route is deprecated
```

Comment:
what is the alternative for these routes? Doesn't the assist app use this route?

Reply:
Klopt de Assist en Capture app gebruiken deze routes maar deze roepen in de App.xaml deze functie aan Infoland.Xamarin.Shared.API.SetFixedApiVersion(2); die ervoor zorgt dat hij altijd versie 2 van de api gebruikt waar de routes nog wel bestaan.